pkg/security/config/config.go

```diff
@@ -22,6 +22,11 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/security/utils"
 )
 
+const (
+	// Minimum value for runtime_security_config.activity_dump.max_dump_size
+	MinMaxDumSize = 10
```

Comment:
I think we should bump this up a bit, 10 feels to small and would be almost the same as 0 tbh. Something like 100 might be enough.

Reply:
Thanks, updated on last commit. :)